src/App.tsx

```diff
@@ -3,33 +3,63 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+interface Movie {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+}
+
+type Props = {};
+
+interface State {
+  films: Movie[],
+  visibleFilms: Movie[] | [],
+  filterBy: string,
```

Comment:
```suggestion
  query: string,
```
не критично, на по условию название такое